src/style.css

```diff
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url("images/star.svg");
+  width: 16px;
+  height: 16px;
+}
+
+.stars--1 div:not(:first-child) {
+  background-image: url("images/star.svg");
+  width: 16px;
+  height: 16px;
+}
+
+.stars--3 div:nth-child(4),
+.stars--3 div:nth-child(5) {
+  background-image: url("images/star.svg");
+  width: 16px;
+  height: 16px;
+}
+
+.stars--4 div:nth-child(5) {
+  background-image: url("images/star.svg");
+  width: 16px;
+  height: 16px;
+}
+
+.stars--2 div:not(:nth-child(1)):not(:nth-child(2)) {
+  background-image: url("images/star.svg");
+  width: 16px;
+  height: 16px;
+}
```

Comment:
A couple of thing to change here:

1.  No need to specify height and width again. You want to put here only the values that are changing,
2. Instead of targeting `div`, use `stars__star` class,
3. As you can see, the values inside of selectors are repeating. Why don't you combine those selectors with comma `,`,
4. There is a better way to specify which child should be affected. Instead of using fixed values like `:nth-child(4)` try to make a use of functional notation. Check it out [in docs](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child)